src/app/register/register.component.scss

```diff
@@ -0,0 +1,131 @@
+@use '@angular/material' as mat;
+@import '../../abstract';
+
+// large parts of this file are copied from either my-account.component.scss for the general layout,
+// or donation-start-form.component.scss for the text input styling.
+// We might want to refactor to remove duplication, but I'm not sure where the common code would fit best,
+// and I think it probably makes sense to wait until we have a third copy and then factor out common parts.
```

Comment:
this is now the third copy so I think the refactor needs to be done, probably before merging this.